src/Yarp.ReverseProxy.Swagger/ReverseProxyDocumentFilter.cs

```diff
@@ -14,17 +16,17 @@ public sealed class ReverseProxyDocumentFilter : IDocumentFilter
     {
         private readonly IHttpClientFactory _httpClientFactory;
         private ReverseProxyDocumentFilterConfig _config;
+        private readonly IConfiguration _configuration;
```

Comment:
I propose to expand the ReverseProxyDocumentFilterConfig by adding Routes so as not to pull the entire configuration.

Reply:
I am not sure from that. When we add to routes there, the developer have to add same route two where while configuring proxy.  Maybe we can read config data from configurationSection

`  public static IReverseProxyBuilder AddSwagger(
            this IReverseProxyBuilder builder,
            IConfigurationSection configurationSection)`

Reply:
You can add public IReadOnlyDictionary<string, RouteConfig> Routes {get; init; } into ReverseProxyDocumentFilterConfig. So you will have Routes which you can use in GetPublishedPaths method.

Reply:
Make sense. Thank you feedback. I have changed it.